homeassistant/components/template/weather.py

```diff
@@ -452,3 +468,252 @@ def _validate_forecast(
                 )
             continue
         return result
+
+
+@dataclass
+class WeatherExtraStoredData(ExtraStoredData):
+    """Object to hold extra stored data."""
+
+    last_temperature: float | None
+    last_humidity: float | None
+    last_wind_speed: float | None = None
+    last_wind_bearing: float | str | None = None
+    last_ozone: float | None = None
+    last_visibility: float | None = None
+    last_pressure: float | None = None
+    last_wind_gust_speed: float | None = None
+    last_cloud_coverage: int | None = None
+    last_dew_point: float | None = None
+    last_apparent_temperature: float | None = None
+    last_forecast: list[Forecast] | None = None
```

Comment:
Why do these have default values? The class is only instantiated in two places, and in both cases all attributes are initialized.